### PR TITLE
Change copy 1.3.7

### PR DESCRIPTION
**Pull request by Tr4pSt3R**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -24,3 +24,5 @@ group :test do
   gem 'simplecov', :require => false
   gem 'sqlite3'
 end
+
+gem 'pry'
```
Comment (neoeno) [inline Gemfile]: Intended? Fine if it is!

Comment (Tr4pSt3R) [inline Gemfile]: @neoeno it was not intended

File: app/views/mortgage_calculator/stamp_duties/_form_step1.html.erb [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <div class="mortgagecalc__panel mortgagecalc__panel--full">
   <div class="intro"><%= I18n.t('stamp_duty.title') %></div>
   <p class="mortgagecalc__intro"><%= I18n.t('stamp_duty.subtitle') %></p>
+  <p class="mortgagecalc__intro">
+     <%= I18n.t('stamp_duty.second_subtitle', :href=>link_to( t('stamp_duty.href_second_subtitle'), t('stamp_duty.url_second_subtitle') ) ).html_safe %>
```
Comment (neoeno) [inline app/views/mortgage_calculator/stamp_duties/_form_step1.html.erb]: Could we do:

```erb
<%= I18n.t('stamp_duty.second_subtitle') %>
<%= link_to t('stamp_duty.href_second_subtitle'), t('stamp_duty.url_second_subtitle') %>
```

That'd save us an `html_safe`. Maybe there's a whitespace concern I'm not aware of tho.

**Comment by neoeno** [comment]
Welsh is absent — guess this is coming through later?

File: app/views/mortgage_calculator/stamp_duties/_form_step1.html.erb (hunk 2 above)
Comment (Tr4pSt3R) [inline app/views/mortgage_calculator/stamp_duties/_form_step1.html.erb]: Good point actually.

**Comment by Tr4pSt3R** [comment]
It was awaiting Welsh translation. That will be going in soon

**Comment by neoeno** [comment]
Unfortunately it fails the frontend tests for that reason! Here's the log:

```rspec
  1) Mortgage Calculator /cy/tools/prynu-ty/cyfrifiannell-treth-stamp 
     Failure/Error: get path
     ActionView::Template::Error:
       translation missing: cy.stamp_duty.href_second_subtitle
     # /Users/caden/Code/mas/mortgage_calculator/app/views/mortgage_calculator/stamp_duties/_form_step1.html.erb:5:in `___sers_caden__ode_mas_mortgage_calculator_app_views_mortgage_calculator_stamp_duties__form_step__html_erb__3278911801861347998_70166737013680'
     # /Users/caden/Code/mas/mortgage_calculator/app/views/mortgage_calculator/stamp_duties/show.html.erb:10:in `___sers_caden__ode_mas_mortgage_calculator_app_views_mortgage_calculator_stamp_duties_show_html_erb___4506802135708978825_70166648714020'
     # ./app/middleware/version_header.rb:13:in `call'
     # ./app/middleware/route_probe.rb:19:in `call'
     # ./app/middleware/override_head.rb:13:in `call'
     # ./app/middleware/capture_request_id.rb:11:in `call'
     # ./spec/requests/tool_integration/mortgage_calculator_spec.rb:12:in `block (4 levels) in <top (required)>'
     # ./spec/spec_helper.rb:81:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:75:in `block (2 levels) in <top (required)>'
```

**Comment by Tr4pSt3R** [comment]
I have received the Welsh translation now so that should be fixed in no time

**Comment by neoeno** [comment]
Neat. With that! :+1:

**Comment by Tr4pSt3R** [comment]
Thanks @neoeno the welsh copy has now been added. Please take another looky :eyes: :eyeglasses:

**Comment by neoeno** [comment]
Looks Welsh to me ;) :+1:

**Comment by Tr4pSt3R** [comment]
Haha Thanks @neoeno

**Comment by dougdroper** [comment]
Yeah, looks good :+1:

**Pull request merged**